flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserBaseSemanticAnalyzer.java

```diff
@@ -1862,6 +1867,65 @@ public static RelNode genValues(
                 rows);
     }
 
+    // traverse the given node to find all correlated variables
+    public static Set<CorrelationId> getVariablesSet(RexNode rexNode) {
```

Comment:
I think we'd better to keep the same name with the Hive. What about renaming to `traverseFilter` and adding the java doc to tell where the code comes from?

Reply:
After reading the `RelOptUtil#getVariablesSet` and `RelOptUtil#getVariablesUsed`, I am fine with the name.